test-files/cases/attribute/unifying/building-3022-duplicate-relation-member-ways-1/Expected.osm

```diff
@@ -55,16 +55,7 @@
     <node visible="true" id="-1" timestamp="1970-01-01T00:00:00Z" version="1" lat="10.0795390480900018" lon="-69.3107096061600032">
```

Comment:
All building test output changes are either a direct result of not creating building relations unless the constituent buildings have 3D tags. Several of the other Attribute Conflation test output changes are a result of adding the overwrite tag exclude list.